Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +7,217 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            
+
+            var firstAccount = peopleAndBalances[0].Split(", ");
+            var firstBalance = Lesson7.StringArrayToFloatArray(firstAccount[1..]);
+
+            var names = new string[] {firstAccount[0]};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5001581)